ddl/db_change_test.go

```diff
@@ -581,14 +581,20 @@ func (s *testStateChangeSuite) testControlParallelExecSQL(c *C, sql1, sql2 strin
 			return
 		}
 		var qLen int64
-		var err1 error
 		for {
 			kv.RunInNewTxn(s.store, false, func(txn kv.Transaction) error {
 				m := meta.NewMeta(txn)
+				// Get the number of jobs from the adding index queue.
+				addIdxLen, err1 := m.DDLJobQueueLen(meta.AddIndexJobListKey)
+				if err1 != nil {
+					return err1
+				}
+
 				qLen, err1 = m.DDLJobQueueLen()
 				if err1 != nil {
 					return err1
 				}
+				qLen += addIdxLen
```

Comment:
Can line 603 increase sleep time?

Reply:
Yes, but I think I don't do it in this PR.